README.md

```diff
@@ -75,6 +75,76 @@ Open a terminal and run the following commands:
 
 </details>
 
+<details>
+
+<summary>Install prerequisites for Linux/WSL2</summary>
+
+# Installing Dependencies on Linux
+
+This guide will walk you through the process of installing the necessary dependencies on a Linux system.
+
+## Prerequisites
+
+Ensure that you have `wget` installed on your system. If not, you can install it using the following command:
+
+```bash
+sudo apt update && sudo apt-get install wget -y
+```
+
+## Steps
+
+1. **Install Microsoft repository**
+
+  ```bash
+  source /etc/os-release
+  wget https://packages.microsoft.com/config/$ID/$VERSION_ID/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+  sudo dpkg -i packages-microsoft-prod.deb
+  rm packages-microsoft-prod.deb
+  ```
+
+2. **Install GitHub Package repository**
+
+  ```bash
+  (type -p wget >/dev/null || (sudo apt update && sudo apt-get install wget -y)) \
+  && sudo mkdir -p -m 755 /etc/apt/keyrings \
+  && wget -qO- https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo tee /etc/apt/keyrings/githubcli-archive-keyring.gpg > /dev/null \
+  && sudo chmod go+r /etc/apt/keyrings/githubcli-archive-keyring.gpg \
+  && echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+  ```
+
+3. **Update packages**
+
+  ```bash
+  sudo apt-get update
+  ```
+
+4. **Install .NET SDK 8.0**
+
+  ```bash
+  sudo apt-get install -y dotnet-sdk-8.0 gh
+  ```
+
+5. **Install Azure CLI**
+
+  ```bash
+  curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+  ```
+
+6. **Install .NET Aspire workload**
+
+  ```bash
+  sudo dotnet workload install aspire
+  ```
+
+7. **Install Yarn**
+
+  ```bash
+  npm install --global yarn
+  ```
```

Comment:
Maybe just collaps 4-7 in one script block so it aliges more with Windows and Mac installations. Preferably also in the same order 

```
sudo apt-get install -y dotnet-sdk-8.0 gh
npm install --global yarn
curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
sudo dotnet workload update && dotnet workload install aspire
```

What about Git, Node, and Docker Desktop? Are they installed by default?

Reply:
Git is default, since it's on WSL docker is default (not sure about pure linux - maybe for a later pr)

Only thing missing is `node`
`sudo apt install nodejs && sudo apt install npm`

(we used `fnm` for installing the correct version etc.)